pkg/controller.v1beta1/suggestion/suggestionclient/suggestionclient.go

```diff
@@ -102,21 +102,22 @@ func (g *General) SyncAssignments(
 		instance.Status.AlgorithmSettings)
 
 	requestSuggestion := &suggestionapi.GetSuggestionsRequest{
-		Experiment:    g.ConvertExperiment(filledE),
-		Trials:        g.ConvertTrials(ts),
-		RequestNumber: int32(requestNum),
+		Experiment:         g.ConvertExperiment(filledE),
+		Trials:             g.ConvertTrials(ts),
+		RequestNumber:      int32(requestNum),
+		TotalRequestNumber: int32(instance.Spec.Requests),
 	}
 
 	// Get new suggestions
 	responseSuggestion, err := rpcClientSuggestion.GetSuggestions(ctx, requestSuggestion)
 	if err != nil {
 		return err
 	}
-	logger.Info("Getting suggestions", "endpoint", endpoint, "response", responseSuggestion,
-		"request", requestSuggestion)
+	logger.Info("Getting suggestions", "endpoint", endpoint, "response", len(responseSuggestion.ParameterAssignments),
+		"requestNum", requestNum)
```

Comment:
Make it more clear here ?

```suggestion
	logger.Info("Getting suggestions", "endpoint", endpoint, "number of response parameters", len(responseSuggestion.ParameterAssignments),
		"number of request parameters", requestNum)
```